Gemfile

```diff
@@ -1,41 +1,37 @@
 source 'https://rubygems.org'
-gem 'rails', '~> 5.0.0', '>= 5.0.0.1'
+gem 'coffee-rails', '~> 4.2'
+gem 'govuk_elements_rails'
+gem 'govuk_frontend_toolkit'
+gem 'govuk_template'
+gem 'high_voltage'
+gem 'jbuilder', '~> 2.5'
+gem 'jquery-rails'
+gem 'lograge'
 gem 'pg', '~> 0.18'
+gem 'pry-rails'
 gem 'puma', '~> 3.0'
+gem 'rails', '~> 5.0.0', '>= 5.0.0.1'
```

Comment:
I think `'>= 5.0.0.1', '< 5.1'` might be a bit more obvious, but that's just aesthetics.